src/components/missions.jsx

```diff
@@ -1,10 +1,19 @@
 /* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
- [ ] it would be better to remove the Eslint disable because Eslint ensures that you write clean code